autoload/ctrlsf/class/line.vim

```diff
@@ -18,7 +18,7 @@ func! ctrlsf#class#line#New(fname, lnum, content) abort
     return {
         \ 'lnum'      : a:lnum,
         \ 'vpos'      : {'normal':{'lnum':-1}},
-        \ 'content'   : a:content,
+        \ 'content'   : substitute(a:content, "\r$", "", ""),
```

Comment:
It's better to fix in `ctrlsf#db#ParseBackendResult`. The result is split into lines by line feed and have no trailing `\r`.

Reply:
You can print debug logs by `let g:ctrlsf_debug_mode=1` and see why substitution is not performed.

Reply:
You will see logs like

```
DefactorizeLine: [Factor]: [/Users/dingye/Dropbox/Projects/CtrlSf/Rakefile, 1, task :archive do], [Orig]: /Users/dingye/Dropbox/Projects/CtrlSf/Rakefile-1-task
```

Please collect them and comment here. I need to confirm my hypothesis that `split` is the root cause for your issue.

Reply:
Ok here is result of search for 'GTool_OT_RayCastEditPins' : 
![image](https://user-images.githubusercontent.com/13521338/175773055-f5eef1dc-fc08-478b-a1b0-01b2df91113e.png)
And after search is performed:
![image](https://user-images.githubusercontent.com/13521338/175773173-c63ac823-48f2-4dc3-b50f-79381f5b1da4.png)
I'm using  `<Plug>CtrlSFCCwordPath<CR>` command for search. Too bad there seem to be no  vimscript dap server for nvim integration.